configs/train/aaf/double_poisson_aaf_cfg.yaml

```diff
@@ -0,0 +1,27 @@
+experiment_name: aaf
+random_seed: 1998
+head_type: double_poisson
+dataset:
+  type: image
+  spec: aaf
+training:
+  accelerator: gpu
+  batch_size: 32
+  chkp_dir: chkp
+  chkp_freq: 6
+  num_epochs: 200
+  precision: bf16-mixed
+  optimizer:
+    type: adam_w
+    kwargs:
+      lr: 0.001
+      weight_decay: 0.00001
+  lr_scheduler:
+    type: cosine_annealing
+    kwargs:
+      T_max: 30
```

Comment:
This should match `num_epochs`. Otherwise you anneal to a basically-zero learning rate by the end of the 30th epoch, which could affect training dramatically